services/api/service.go

```diff
@@ -660,9 +660,9 @@ func (api *RelayAPI) RespondError(w http.ResponseWriter, code int, message strin
 	}
 }
 
-func (api *RelayAPI) RespondOK(w http.ResponseWriter, response any) {
```

Comment:
i wonder if we should keep `RespondOK` around and just introduce `Respond` (which both RespondOK and RespondError could use)?

Reply:
added back RespondOK and simplified!

Reply:
looks great